src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -31,24 +33,30 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
         // --- end of chromosome chart ---
 
         // --- CNA bar chart ---
-        let cnaRaphaelData: any = {};
-        cnaRaphaelData[sampleId] = [];
-        _.each(this.props.cnaSegments, function(_dataObj: any) {
-            var _tmp: Array<any> = [];
-            _tmp.push(_dataObj.sample);
-            _tmp.push(_dataObj.chr);
-            _tmp.push(_dataObj.end);
-            _tmp.push(_dataObj.start);
-            _tmp.push(_dataObj.numProbes);
-            _tmp.push(_dataObj.value);
-            cnaRaphaelData[sampleId].push(_tmp);
+        _.each(uniqCnasampleIds, (_sampleId: string) => {
+            let raphaelData: Array<any> = [];
+            var _trackData = _.filter(this.props.cnaSegments, function(_cnaObj: any) { return _cnaObj.sample === _sampleId; });
```

Comment:
the _cnaObj here can't really map to the type, I'll have to wait till the end point is implemented. Will add a TODO here.